doc/users/dflt_style_changes.rst

```diff
@@ -1065,6 +1065,17 @@ Z-order
 
 
 
+``ScalarFormatter`` tick label formatting with offsets
+======================================================
+
+With the default of ``rcParams['axes.formatter.useoffset'] = True``,
+an offset will be used when it will save 4 or more digits.  This can
```

Comment:
No worrys, as it sounds complicated. Is there a typical use case for this sort of offsetting or a name for it? (Like it's not scientific notation, which is also partly why it's so confusing)

It seems like it encodes ticks to reduce their size by offset or more digits.